charts/osm/crds/multiclusterservice.yaml

```diff
@@ -41,6 +41,9 @@ spec:
                 - serviceAccount
                 - clusters
               properties:
+                globalIP:
+                  type: string
+                  description: globalIP is the IP address for the implicit global service which references every defined cluster. It is usually assigned at random by OSM.
```

Comment:
> for the implicit global service

This is not clear to me. What is this referring to? Public IP? Internal k8s service IP?

Reply:
Internal k8s service IP.. it will need to be created by a mutating webhoook

Reply:
Why would the service IP need to be created by a mutating webhook? Would you mind adding a brief section on how this is going to be leveraged in the design doc.

Reply:
yup, will do!

would it be possible to do that async and continue with this PR?